src/server/stories.ts

```diff
@@ -20,17 +20,18 @@ import { isStorybookVersionLessThan } from './storybook/helpers';
 function storyTestFabric(delay?: number, testFn?: CreeveyTestFunction) {
   return async function storyTest(this: Context) {
     delay ? await new Promise((resolve) => setTimeout(resolve, delay)) : void 0;
-    await (testFn?.call(this) ?? this.screenshots.length > 0
-      ? this.expect(
-          this.screenshots.reduce(
-            (screenshots, { imageName, screenshot }, index) => ({
-              ...screenshots,
-              [imageName ?? `screenshot_${index}`]: screenshot,
-            }),
-            {},
-          ),
-        ).to.matchImages()
-      : this.expect(await this.takeScreenshot()).to.matchImage());
+    await (testFn?.call(this) ??
```

Comment:
I don't know why are we expecting return value from a test function. I think we should check if there is a testFn and if it's then call it

Reply:
Fixed it.